GANDLF/models/dynunet_wrapper.py

```diff
@@ -2,6 +2,42 @@
 import monai.networks.nets.dynunet as dynunet
 
 
+def get_kernels_strides(sizes, spacings):
+    """
+    More info: https://github.com/Project-MONAI/tutorials/blob/main/modules/dynunet_pipeline/create_network.py#L19
+
+    When refering this method for other tasks, please ensure that the patch size for each spatial dimension should
+    be divisible by the product of all strides in the corresponding dimension.
+    In addition, the minimal spatial size should have at least one dimension that has twice the size of
+    the product of all strides. For patch sizes that cannot find suitable strides, an error will be raised.
+
+    """
+    input_size = sizes
+    strides, kernels = [], []
+    while True:
+        spacing_ratio = [sp / min(spacings) for sp in spacings]
+        stride = [
+            2 if ratio <= 2 and size >= 8 else 1
+            for (ratio, size) in zip(spacing_ratio, sizes)
+        ]
+        kernel = [3 if ratio <= 2 else 1 for ratio in spacing_ratio]
+        if all(s == 1 for s in stride):
+            break
+        for idx, (i, j) in enumerate(zip(sizes, stride)):
+            if i % j != 0:
+                raise ValueError(
+                    f"Patch size is not supported, please try to modify the size {input_size[idx]} in the spatial dimension {idx}."
```

Comment:
I would suggest converting this to an `assert` (for consistency across the codebase):
```suggestion
            assert i % j == 0, f"Patch size is not supported, please try to modify the size {input_size[idx]} in the spatial dimension {idx}."
```

Reply:
Yes, you are right.  :P